filter/common.c

```diff
@@ -348,26 +348,23 @@ WriteLabelProlog(const char *label,	/* I - Page label */
       }
   }
 
-  if (label)
-  {
-    if (classification[0])
-      printf(" - ");
+  if (classification[0])
+    printf(" - ");
 
    /*
     * Quote the label string as needed...
     */
 
-    for (ptr = label; *ptr; ptr ++)
-      if (*ptr < 32 || *ptr > 126)
-        printf("\\%03o", *ptr);
-      else
-      {
-	if (*ptr == '(' || *ptr == ')' || *ptr == '\\')
-	  putchar('\\');
+  for (ptr = label; *ptr; ptr++)
+    if (*ptr < 32 || *ptr > 126)
```

Comment:
For consistency with the current coding guidelines, we now would put braces around each of these blocks. So braces for the `for` loop and for this `if` statement (don't need them for the nested `if` below because there are no compound statements).